package.json

```diff
@@ -36,9 +36,11 @@
   "author": "eight04 <eight04@gmail.com>",
   "license": "MIT",
   "devDependencies": {
+    "@rollup/plugin-commonjs": "^26.0.1",
     "c8": "^10.1.2",
     "endent": "^2.1.0",
     "eslint": "^9.5.0",
+    "globals": "^15.8.0",
```

Comment:
What does `globals` do?

Reply:
Because you imported it in eslint.config.mjs. transitive dependencies is not work in pnpm.
I've deleted it.